Tensile/SolutionStructs.py

```diff
@@ -2332,6 +2336,41 @@ def checkAndAssignWaveSeparateGlobalRead(state, tc):
         reject(state, "dind't support WaveSeparateGlobalRead when MacroTile is not multiple of wave %u in TLU%s" % (state["MacroTile%s"%tc], tc))
 
 
+  ########################################
+  # determine can we use DirectToVgpr
+  @staticmethod
+  def isDirectToVgprDoable(state, tc):
+    # With MatricInstruction only (tentative)
```

Comment:
Matrix not Matric, also still tentative or can that be removed?

Reply:
Changed to Matrix...